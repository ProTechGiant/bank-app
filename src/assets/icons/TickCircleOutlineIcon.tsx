import { Path, Svg } from "react-native-svg";

import { IconProps } from "@/assets/icons";

export function TickCircleOutlineIcon({ color = "#FAFAFA", height = 14, width = 15 }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 12 13" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M6 1.5C3.24 1.5 1 3.74 1 6.5C1 9.26 3.24 11.5 6 11.5C8.76 11.5 11 9.26 11 6.5C11 3.74 8.76 1.5 6 1.5ZM6 10.5C3.795 10.5 2 8.705 2 6.5C2 4.295 3.795 2.5 6 2.5C8.205 2.5 10 4.295 10 6.5C10 8.705 8.205 10.5 6 10.5ZM8.295 4.29L5 7.585L3.705 6.295L3 7L5 9L9 5L8.295 4.29Z"
        fill={color}
      />
    </Svg>
  );
}
