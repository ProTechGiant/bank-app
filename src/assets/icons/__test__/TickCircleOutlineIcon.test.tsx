import { create } from "react-test-renderer";

import { TickCircleOutlineIcon } from "../TickCircleOutlineIcon";

describe("icon/TickCircleOutlineIcon", () => {
  it("renders a TickCircleOutlineIcon", () => {
    const result = create(<TickCircleOutlineIcon />);

    expect(result).toMatchSnapshot();
  });
});
