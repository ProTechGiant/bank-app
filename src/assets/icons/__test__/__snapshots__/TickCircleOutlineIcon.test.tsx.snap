// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/TickCircleOutlineIcon renders a TickCircleOutlineIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="14"
  bbWidth="15"
  fill="none"
  focusable={false}
  height={14}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 14,
        "width": 15,
      },
    ]
  }
  vbHeight={13}
  vbWidth={12}
  width={15}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M6 1.5C3.24 1.5 1 3.74 1 6.5C1 9.26 3.24 11.5 6 11.5C8.76 11.5 11 9.26 11 6.5C11 3.74 8.76 1.5 6 1.5ZM6 10.5C3.795 10.5 2 8.705 2 6.5C2 4.295 3.795 2.5 6 2.5C8.205 2.5 10 4.295 10 6.5C10 8.705 8.205 10.5 6 10.5ZM8.295 4.29L5 7.585L3.705 6.295L3 7L5 9L9 5L8.295 4.29Z"
      fill={
        {
          "payload": 4294638330,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
